masonry-layout/masonry-layout.d.ts

```diff
@@ -0,0 +1,61 @@
+// Type definitions for Masonry 4.0.0
+// Project: https://github.com/desandro/masonry
+// Definitions by: Travis Brown <https://github.com/warriorrocker>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../jquery/jquery.d.ts" />
+
+declare class Masonry implements IMasonry {
+	constructor(options?: IMasonryOptions);
+	constructor(selector: string, options?: IMasonryOptions);
+}
+
+interface IMasonry {
```

Comment:
http://www.typescriptlang.org/docs/handbook/writing-definition-files.html
> Naming Conventions
In general, you shouldn’t prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.